tests/lightning_qubit/test_adjoint_jacobian_class.py

```diff
@@ -23,11 +23,23 @@
 from pennylane.tape import QuantumScript
 from scipy.stats import unitary_group
 
-from pennylane_lightning.lightning_qubit._adjoint_jacobian import LightningAdjointJacobian
-from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+if device_name == "lightning.qubit":
+    from pennylane_lightning.lightning_qubit._adjoint_jacobian import LightningAdjointJacobian
+    from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
 
-if device_name != "lightning.qubit":
-    pytest.skip("Exclusive tests for lightning.qubit. Skipping.", allow_module_level=True)
+if device_name == "lightning.kokkos":
+    from pennylane_lightning.lightning_kokkos._adjoint_jacobian import (
+        LightningKokkosAdjointJacobian as LightningAdjointJacobian,
+    )
+    from pennylane_lightning.lightning_kokkos._state_vector import (
+        LightningKokkosStateVector as LightningStateVector,
+    )
+
+if device_name not in ("lightning.qubit", "lightning.kokkos"):
+    pytest.skip(
+        "Exclusive tests for new API backends lightning.qubit and lightning.kokkos for LightningAdjointJacobian class. Skipping.",
+        allow_module_level=True,
+    )
```

Comment:
Lightning devices, at least for now, have an [attribute](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/lightning_qubit/lightning_qubit.py#L478) tracking if this is a new_API device.
This should work:
```suggestion
if not LightningDevice._new_API:
    pytest.skip(
        "Exclusive tests for new API backends LightningAdjointJacobian class. Skipping.",
        allow_module_level=True,
    )
```